pudl/transform/ferc1.py

```diff
@@ -338,29 +351,45 @@ def plants_steam(ferc1_raw_dfs, ferc1_transformed_dfs):
     # them a FERC Plant ID, and pull the results back out into a dataframe:
     plants_w_ids = pd.DataFrame()
     for plant_id_ferc1, plant in enumerate(ferc_plants):
-        nx.set_edge_attributes(plant, plant_id_ferc1 +
-                               1, name='plant_id_ferc1')
+        nx.set_edge_attributes(plant,
+                               plant_id_ferc1 + 1,
+                               name='plant_id_ferc1')
         new_plant_df = nx.to_pandas_edgelist(plant)
         plants_w_ids = plants_w_ids.append(new_plant_df)
     logger.info(
         f"Successfully Identified {plant_id_ferc1+1-len(orphan_record_ids)} "
         f"multi-year plant entities.")
-
-    # Ultimately we just want a record_id to plant_id_ferc1 mapping, so we can
-    # ignore the target record_id now:
-    plants_w_ids = plants_w_ids.drop('target', axis=1).drop_duplicates()
-    plants_w_ids = plants_w_ids.rename({'source': 'record_id'}, axis=1)
-    # This is just so we can look at the results easily.
-    plants_w_ids = plants_w_ids.sort_values(['plant_id_ferc1', 'record_id'])
-
+    # Now we need a list of all the record IDs, with their associated
```

Comment:
Is this the section where you made the bulk of the changes/improvements?

Reply:
Yes, previously I was just looking at one end of the edge (I think it was `source`) which it turns out wasn't a complete list of the involved nodes. So this building up of the full `source` and `target` list of nodes was the main change.

Reply:
okay great. that makes sense to me. It seems clear but it might be nice to add a comment for each of the elements to remind our future selves exactly why we're compiling each thing.